src/components/slat/AposSlatList.vue

```diff
@@ -0,0 +1,106 @@
+
+<template>
+  <draggable class="apos-slat-list" tag="ul" v-model="items" 
+    v-bind="dragOptions" :move="onMove" @start="isDragging=true" 
+    @end="isDragging=false"
+  >
+    <transition-group type="transition" :name="'flip-list'">
```

Comment:
`:name="'flip-list'"` is doing more than it needs to. `name="flip-list"` should be enough since you're simply using a string.

Reply:
I can't tell if `flip-list` is doing anything different from simply `list`. Do you know it is?